.asf.yaml

```diff
@@ -0,0 +1,85 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+github:
+  description: One of the most advanced and mature open-source MPP (Massively Parallel Processing) databases available. Open source alternative to Greenplum Database.
+  homepage: https://cloudberry.apache.org
+  labels:
+    - mpp
+    - big-data
+    - data-warehouse
+    - data-analysis
+    - olap
+    - distributed-database
+    - database
+    - postgres
+    - postgresql
+    - greenplum
+    - ai
+    - sql
+    - c
+  features:
+    # Enable wiki for documentation
+    wiki: false
+    # Enable issues management
+    issues: true
+    # Enable projects for project management boards
+    projects: true
+  enabled_merge_buttons:
+    # enable squash button:
+    squash: true
+    # disable merge button:
+    merge: false
+    # enable rebase button:
+    rebase: true
+  protected_branches:
+    main:
+      required_status_checks:
+        # strict means "Require branches to be up to date before merging".
+        strict: true
+        # contexts are the names of checks that must pass.
+        contexts:
+          - cbdb pipeline/build
+          - cbdb pipeline/icw-test
+          - cbdb pipeline/ic-singlenode-test
+          - cbdb pipeline/icw-expandshrink-test
+          - cbdb pipeline/icw-resgroup-test
+          - cbdb pipeline/icw-orca-test
+          - cbdb pipeline/icw-parallel-test
+          - cbdb external_fts pipeline/build
+          - cbdb external_fts pipeline/icw-test
+      required_pull_request_reviews:
+        dismiss_stale_reviews: true
+        required_approving_review_count: 2
+      # squash or rebase must be allowed in the repo for this setting to be set to true.
+      required_linear_history: true
+
+      required_signatures: false
+
+      # requires all conversations to be resolved before merging is possible
+      required_conversation_resolution: true
+  del_branch_on_merge: true
+  dependabot_alerts: true
+  dependabot_updates: false
+  protected_tags:
+    - "[0-9]*.*"
+notifications:
+  commits: commits@cloudberry.apache.org
+  issues: dev@cloudberry.apache.org
```

Comment:
This could lead to heavy traffic in dev and cause subscribers to suffer from spam.

It might be beneficial to keep discussions on issues and pull requests private between authors and commentators. If they cannot reach a consensus there, then they can decide to raise another thread in the dev

Reply:
Hi @yaooqinn Thank you for your feedback. It is very helpful!